packages/app/src/templates/template.container.tsx

```diff
@@ -5,19 +5,22 @@
 
 import { connect } from "react-redux"
 import { ReduxRootState } from "Renderer/store"
+import { deleteTemplates } from "App/templates/actions"
 import { Templates } from "App/templates/components"
 import { createTemplate } from "App/templates/actions"
-import { deleteTemplates } from "App/templates/actions"
+import { hideDeleteModal } from "App/templates/actions/base.action"
```

Comment:
Shouldn't `hideDeleteModal` be added to `index.ts` so it could be imported from `App/templates/actions`? Not 100% sure, just asking.

Reply:
`hideDeleteModal` was already added to index. Import path changed in [7c222ee](https://github.com/mudita/mudita-center/pull/764/commits/7c222eed5a12a682f3d9b7d9ec91c06d8a7edc54)